app/presenters/iiif_print/work_show_presenter_decorator.rb

```diff
@@ -24,13 +24,20 @@ def iiif_viewer?
     #
     # @todo Review if this is necessary for Hyrax 5.
     def members_include_viewable_image?
-      all_member_ids = solr_document.try(:member_ids) || solr_document.try(:[], 'member_ids_ssim')
+      all_member_ids = load_file_set_ids(solr_document)
       Array.wrap(all_member_ids).each do |id|
         return true if file_type_and_permissions_valid?(member_presenters_for([id]).first)
       end
       false
     end
 
+    def load_file_set_ids(solr_doc)
```

Comment:
we could extract this method into a module since I think it's the same one declared in app/presenters/iiif_print/iiif_manifest_presenter_factory_decorator.rb#L25

Reply:
Agreed... I changed an existing method which was also duplicated. To save time waiting for another build now, maybe we can do it when we do the other ticket about going directly to the first page found?